### PR TITLE
getting started with tasks: update log messages to use latest version

### PR DESCRIPTION
**Pull request by clintonsteiner**
# Changes

/kind documentation

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: content/en/docs/Getting-started/tasks.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ This tutorial shows you how to
 1.  [Install minikube](https://minikube.sigs.k8s.io/docs/start/). You only have
     to complete the step 1, "Installation".
 
+1.  Ensure docker is installed and running.
```
Comment (AlanGreene) [inline content/en/docs/Getting-started/tasks.md:23]: Thanks for the PR @clintonsteiner.

This is already covered in the minikube install instructions linked above:

> What you’ll need
> …
> Container or virtual machine manager, such as: **[Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/)**, [QEMU](https://minikube.sigs.k8s.io/docs/drivers/qemu/), [Hyperkit](https://minikube.sigs.k8s.io/docs/drivers/hyperkit/), [Hyper-V](https://minikube.sigs.k8s.io/docs/drivers/hyperv/), [KVM](https://minikube.sigs.k8s.io/docs/drivers/kvm2/), [Parallels](https://minikube.sigs.k8s.io/docs/drivers/parallels/), [Podman](https://minikube.sigs.k8s.io/docs/drivers/podman/), [VirtualBox](https://minikube.sigs.k8s.io/docs/drivers/virtualbox/), or [VMware Fusion/Workstation](https://minikube.sigs.k8s.io/docs/drivers/vmware/)

Comment (clintonsteiner) [reply content/en/docs/Getting-started/tasks.md:23]: Ah - I didn't drill into the minikube instructions
I'm fine leaving it out then

Comment (clintonsteiner) [reply content/en/docs/Getting-started/tasks.md]: Rebased to just push the update to the logs

**Review by AlanGreene — APPROVED** [review]
Thanks @clintonsteiner 
/lgtm

**Pull request merged**